res/authors.txt

```diff
@@ -43,6 +44,7 @@ quest_icons.svg
 	notes
 	parking
 	parking_maxstay
+    fire_hydrant               modified from Creative Tail
```

Comment:
Not all icons from creative trail are free. You need to refer to the specific icon so that it is clear where it is coming from and that the license is correct